mobile-app/src/screens/Profile/Profile.js

```diff
@@ -2,30 +2,50 @@ import React, { Component } from 'react';
 import { View, Image, Text, ActivityIndicator } from 'react-native';
 import { compose } from 'recompose';
 import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
 
 import { Divider } from '../../components/Divider';
 import { selectors as userSelectors } from '../../reducers/user';
+import { operations as teamsOps } from '../../reducers/teams';
+import { operations as userOps } from '../../reducers/user';
 import { selectors as appSelectors } from '../../reducers/app';
 import { withNavigationHelpers } from '../../services/Navigation';
 import styles from './styles';
-
+import { COUNTRIES_HASH } from '../../shared/countries';
+import SelectBlock from './SelectBlock';
 
 class Profile extends Component {
+
+  componentWillMount() {
+    this.props.getProfile();
+  }
+
+  handleTeamPress = team => {
+    this.props.navigation.navigate('TeamProfile', { team });
```

Comment:
hardcode

Reply:
@KucherenkoIhor 
Everywhere in application navigation happens like here.
Do you want me to create some const ROUTES and use it to navigate between screens like this?
`this.props.navigation.navigate(ROUTES.TEAM_PROFILE, { team });`

Reply:
Yap